tests/merge_base_class_test.py

```diff
@@ -0,0 +1,32 @@
+import ipaddress
+import pandas as pd
+from kartograf.merge import BaseNetworkIndex
+
+
+def test_base_dict_create():
+    base = BaseNetworkIndex()
+    state = base.get()
+    assert state == {4: {}, 6: {}}
```

Comment:
nit: This is a bit pedantic but I don't think it's great testing style to expose the internals. Ideally the tests would check the internal through the interface we need and we wouldn't need a special function (get) to expose the internals. It's fine for me to keep it for now but I would consider it a code smell if we can not refactor the internals without changing the test. My alternative suggestion would be to randomly check 10 keys of IPv4 and 10 keys of the IPv6 dict via the inclusion function and ensure that no result is ever returned. For IPv4 it could even be all 255 and wouldn't take much time either.

Reply:
not pedantic, thanks! I modified the test to check for one IPv4 row, the logic is equally valid for 1 or 10 rows. Still need to expand it to cover IPv6.